pkg/operator/controller/prometheus.go

```diff
@@ -147,7 +147,7 @@ func getAlertRules(runbookURLTemplate string) []promv1.Rule {
 	return []promv1.Rule{
 		generateAlertRule(
 			"CDIOperatorDown",
-			"kubevirt_cdi_operator_up_total == 0",
+			"kubevirt_cdi_operator_up == 0",
```

Comment:
@akalenyu @avlitman This metric's value can be different than 0/1, right?
(Expr: `sum(up{namespace='%s', pod=~'cdi-operator-.*'} or vector(0))`)
If so, I think `kubevirt_cdi_operator_up` may be a bit misleading, as it sounds like a boolean metric.